student/serializers/__init__.py

```
@@ -0,0 +1,3 @@
+from .auth import LogInSerializer
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)